### PR TITLE
Add remaining OIDC docs

### PR DESCRIPTION
**Pull request by sherenewso2**

### CODE REVIEW

File: en/docs/fragments/oauth-oidc-grants.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+{!fragments/register-a-service-provider.md!}
+
+3. Expand **Inbound Authentication Configuration** section and then **OAuth/OpenID Connect Configuration**.
```
Comment (gomathyk) [inline en/docs/fragments/oauth-oidc-grants.md:3]: the **Inbound Authentication Configuration** section

File: en/docs/fragments/oauth-oidc-grants.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    scope=openid
+    ```
+
+    You will receive an authorization code. 
+
+    ```
+    http://wso2is.local:8080/playground2/oauth2client?code=9142d4cad58c66d0a5edfad8952192
+    ```
+
+2. Run the following curl command using the authorization code recieved, to obtain the access token.
```
Comment (gomathyk) [inline en/docs/fragments/oauth-oidc-grants.md:51]: received

File: en/docs/guides/access-delegation/map-oidc-scopes-claims.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# Map OpenID Connect Scopes and Claims
+
+This page guides you through mapping [OpenID Connect (OIDC) scopes and claims](../../../concepts/authentication/scopes-claims). 
+
+**About oidc-scope-config.xml**
+
+The `oidc-scope-config.xml` file enables grouping of claims that are bound to a scope value in OpenID Connect (OIDC). By default, the oidc-scope-config.xml file is located inside `<IS_HOME>/repository/conf/identity` directory.
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/map-oidc-scopes-claims.md:7]: inside the `<IS_HOME>/repository/conf/identity` directory.

File: en/docs/guides/access-delegation/map-oidc-scopes-claims.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    
+This file can be modified to add custom scopes or claims. When the server starts, these configurations are stored in the registry located in `/oidc/` . You can customize claims or scopes from here as well. The returned clams from id token or user info endpoint will be decided based on both the requested scopes and requested claims.
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/map-oidc-scopes-claims.md:9]: returned claims from ID token

File: en/docs/guides/access-delegation/map-oidc-scopes-claims.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    
+When requesting for an OIDC token, you can specify a scope value that is bound to a set of claims in the `oidc-scope-config.xml` file. When sending that OIDC token to the userinfo endpoint, only the claims that are common to both the `oidc-scope-config.xml` and the service provider claim configuration, will be returned.
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/map-oidc-scopes-claims.md:11]: `oidc-scope-config.xml` file

File: en/docs/guides/access-delegation/map-oidc-scopes-claims.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+-----
+
+## Add scopes
+
+1. Log in to the [Management Console](insertlink) using admin/admin credentials. 
+
+2. Under **Manage** click **OIDC Scopes > Add**.
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/map-oidc-scopes-claims.md:19]: **Manage**,

File: en/docs/guides/access-delegation/map-oidc-scopes-claims.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+5. Click **Finish** to add the new scope claim mapping.
+
+-----
+
+## Edit or list scopes
+
+1. Log in to the [Management Console](insertlink) using admin/admin credentials. 
+
+2. Under **Manage** click **OIDC Scopes > List**.
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/map-oidc-scopes-claims.md:35]: Under **Manage*,

File: en/docs/guides/access-delegation/oidc-hybrid-flow.md [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+    ```
+
+4. If there are two id\_tokens issued, where one id\_token is from authorization endpoint and other is from token endpoint, be sure to perform the [neccessary validations](#validations), which are based on the OpenID Connect specification.
+
+----
+
+## id_token validations
+
+If there are two id\_tokens issued, where one id\_token is from authorization endpoint and other is from token endpoint, be sure to perform the following validations based on the OpenID Connect specification.
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/oidc-hybrid-flow.md:218]: It's a little odd to have the same sentence as a step in the previous topic and then as the intro here. Shall we remove/modify this?

File: en/docs/guides/access-delegation/oidc-hybrid-flow.md [hunk 9]
```diff
@@ -0,0 +1,227 @@
...
+
+1.  Ensure that the `iss` and `sub` claim values are identical in the two `id_tokens` .
+
+2.  If any one of the id tokens contain claims about the end user, and are present in both, the values of the claims should be the same in both.
+
+3.  All claims about the authentication event that is present in either should be present in both.
+
+4.  The `at\_hash` and `c\_hash` claims may be omitted from the id token returned from the token endpoint even when the claims are present in
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/oidc-hybrid-flow.md:226]: ID token

File: en/docs/guides/access-delegation/oidc-logout-url-redirection.md [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+
+However, there can be instances where you cannot use a GET request as the OIDC logout request. For example, when the generated id_token exceeds the maximum character limit of the URL, and the browser truncates the URL. In such instances, you can send the logout request as a POST request using an automatically submitting HTML form.
+
+-----
+
+## Sending a GET logout request
+
+{!fragments/register-a-service-provider.md!}
+
+3. Expand **Inbound Authentication Configuration** section and then **OAuth/OpenID Connect Configuration**.
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/oidc-logout-url-redirection.md:15]: Expand the

File: en/docs/guides/access-delegation/oidc-logout-url-redirection.md [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+
+    ```toml
+    [authentication]
+    sign_auth_response_with_tenant_of= "sp" #user
+    ```
+
+    If it is set to `user`, expand **Local & Outbound Authentication Configuration** on the management console service provider configuration and select **Use tenant domain in local subject identifier**. 
+
+
+8. Use the following cURL command to retrieve the `id_token` using the client id, client secret, and authorization code.
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/oidc-logout-url-redirection.md:46]: ID

**Comment by sherenewso2** [comment]
Will do the requested changes in the next PR as there is a bit of a conflict with my current clone @gomathyk

**Pull request merged**